cmd/list/instancetypes/cmd.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package instancetypes
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/ocm/machines"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "instance-types",
+	Aliases: []string{"instanceTypes"},
+	Short:   "List Instance types",
+	Long:    "List Instance types that are available.",
```

Comment:
Think we need some additional qualifier here?  "are available for use with ROSA"